packages/xrpl/test/connection.test.ts

```diff
@@ -68,250 +136,366 @@ describe('Connection', function () {
       originalConsoleLog = console.log
     })
 
-    afterEach(function () {
+    afterEach(async () => {
```

Comment:
ditto

Reply:
Don't have to be, consistent with other files